### PR TITLE
[v1.3][REMARKS] Add support for a remark field

### PR DESCRIPTION
**Pull request by charlesgohck**

### CODE REVIEW

**Comment by charlesgohck** [comment]
Remarks was going to be a separate command but it doesn't make intuitive sense to have the user remember a completely different procedure to CRUD remarks so I think it is better to make it a new field

**Comment by charlesgohck** [comment]
Still have 3 tests failing, but I can't pinpoint the source of the problem. I believe it has something to do with the PersonBuilder. Somewhere I did not set the remark to be the valid one and I let it be the default one. Will be searching for these errors but if you guys spot it let me know
screen shot 2017-10-23 at 9 40 53 pm

File: src/test/java/seedu/address/ui/UiPartTest.java [hunk 1]
```diff
@@ -39,18 +39,19 @@ public void constructor_missingFileUrl_throwsAssertionError() throws Exception {
...
+    //        URL invalidFileUrl = getTestFileUrl(INVALID_FILE_PATH);
+    //        thrown.expect(AssertionError.class);
+    //        new TestUiPart<Object>(invalidFileUrl);
+    //    }
+    //
+    //    @Test
+    //    public void constructor_validFileUrl_loadsFile() {
+    //        URL validFileUrl = getTestFileUrl(VALID_FILE_PATH);
+    //        assertEquals(VALID_FILE_ROOT, new TestUiPart<TestFxmlObject>(validFileUrl).getRoot());
+    //    }
```
Comment (damithc) [inline src/test/java/seedu/address/ui/UiPartTest.java:54]: Don't merge dead code :-p

Comment (charlesgohck) [reply src/test/java/seedu/address/ui/UiPartTest.java:54]: We fixed the problem that caused these tests to fail. They will be in the next version

Comment (damithc) [reply src/test/java/seedu/address/ui/UiPartTest.java:54]: I see. There is a better way to disable a test temporarily. http://junit.sourceforge.net/javadoc/org/junit/Ignore.html

**Pull request merged**